doc/api.rst

```diff
@@ -29,6 +29,7 @@ Models
    SparseLogreg
    SVM
    SVR
+   SimplexSVR
```

Comment:
nitpick: do we put them in alphabetical order ?

Reply:
you're right. It's been changed